python/lsst/ip/isr/isrTask.py

```diff
@@ -1071,13 +1074,20 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
 
         # Load the correct style of brighter-fatter kernel, and repack
         # the information as a numpy array.
+        brighterFatterSource = None
         if self.config.doBrighterFatter:
             brighterFatterKernel = inputs.pop('newBFKernel', None)
             if brighterFatterKernel is None:
                 # This type of kernel must be in (y, x) index
                 # ordering, as it used directly as the .array
                 # component of the afwImage kernel.
                 brighterFatterKernel = inputs.get('bfKernel', None)
+                brighterFatterSource = 'bfKernel'
+                additionalInputDates[brighterFatterSource] = self.extractCalibDate(brighterFatterKernel)
+
+            if brighterFatterKernel is None:
+                # This was requested by the config, but none were found.
+                raise RuntimeError("No brighter-fatter kernel was supplied.")
 
             if brighterFatterKernel is not None and not isinstance(brighterFatterKernel, numpy.ndarray):
```

Comment:
Can't this just be `elif not isinstance(brighterFatterKernel, numpy.ndarray)` now?